std/format.d

```diff
@@ -7085,6 +7130,8 @@ if (is(T == float) || is(T == double) || (is(T == real) && T.mant_dig == double.
             return printFloatA(val, f, rm, sgn, exp, mnt, is_upper);
         case 'e': case 'E':
             return printFloatE(val, f, rm, sgn, exp, mnt, is_upper);
+        case 'f': case 'F':
+            return printFloatF(val, f, rm, sgn, exp, mnt, is_upper);
```

Comment:
Following the success of the float zero factorisation, I'm beginning to think that the best way to organise the printing is not by format specifier (%a,%f,%g,%e) but by "category" (for want of a better term), i.e. 
* inf/nan (handled above), 
* Algorithm A: For large exponents (exp >= T.mant_dig)
*  Algorithm B: For small exponents (exp < T.mant_dig - 61)
* Algorithm C: For exponents close to 0. (already done)

Since there seems to be much more in common between the different ranges of numbers than there is between the different format specifiers.